modules/telco-core-workloads-on-schedulable-control-planes.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="telco-core-workloads-on-schedulable-control-planes_{context}"]
+= Workloads on schedulable control planes
+
+Enabling workloads on control plane nodes::
+
+You can enable schedulable control planes to run workloads on control plane nodes, utilizing idle CPU capacity on bare-metal machines for potential cost savings. This feature is only applicable to clusters with bare-metal control plane nodes.
++
+There are two distinct parts to this functionality: allowing workloads on control plane nodes, which can be configured after initial installation, and enabling workload partitioning, which must be done during initial day zero cluster installation. The supported configuration requires both to be enabled at the same time during the cluster's initial setup. Enabling workload partitioning helps to isolate the control plane from regular workloads, ensuring the cluster remains stable and reliable.
```

Comment:
May need clarification here. It sounds like sched control planes needs to be enabled during initial installation which I don't think is true. Workload Partitioning _**must**_ be enabled at install time but I believe sched ctrl planes can be enabled later (please confirm @MarSik)

Reply:
Correct.

Reply:
The first sentence says it correctly, the problematic is just the second one with the "at the same time".

Reply:
I rewrote section as was unclear and gave incorrect instruction.